src/core/adapters/SimplePoolAdapter.ts

```diff
@@ -140,13 +139,11 @@ export abstract class SimplePoolAdapter implements IProtocolAdapter {
     input: GetTotalValueLockedInput,
   ): Promise<ProtocolTotalValueLockedToken[]>
 
-  async getOneDayProfit({
+  async getProfits({
     userAddress,
-    blockNumber,
+    toBlock,
+    fromBlock,
   }: GetProfitsInput): Promise<ProfitsTokensWithRange> {
-    const toBlock = blockNumber
-    const fromBlock = toBlock - AVERAGE_BLOCKS_PER_DAY[this.chainId]
-
```

Comment:
block numbers now gets passed into function which feels cleaner

Reply:
This is going to be a massive benefit for the integration tests too as  it will allow consistency even if we update the average blocks per month. I like it.